groovy/gradle.javaee/licenseinfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/gradle/javaee/resources/WebPagesBadge.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/gradle/javaee/resources/javaeeProjectIcon.png</file>
+        <file>src/org/netbeans/modules/gradle/javaee/resources/javaeeProjectIcon_dark.png</file>
+        <license ref="Apache-2.0" />
+        <!-- The complete content of the NOTICE file from: https://raw.githubusercontent.com/gradle/gradle/v6.0.0/subprojects/distributions/src/toplevel/NOTICE -->
```

Comment:
I think this can be completely removed. From my earlier comment 

> For the icons I see nothing that is relevant in the notice file.

From the [Apache License 2.0 (section 4d)](https://www.apache.org/licenses/LICENSE-2.0#redistribution):

> [...] must include a readable copy of the attribution notices contained within such NOTICE file, excluding those notices that do not pertain to any part of the Derivative Works, in at least one [...]

The NOTICE file has neither a general attribution, nor anything related to artwork. So I claim, that we can drop it completely (this may be worth an XML comment in the file).

This comments applies to all `licenseinfo.xml` files.